npm/react/plugins/utils/wrap-devserver.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function returnInjectDevServerFunction (devServer, postPocessConfig = () => {}) {
```

Comment:
Why not call this `wrapDevServer` instead of `returnInjectDevServerFunction`?

Reply:
I used the term I found in other places in the codebase.
I prefer yours.

https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/plugins/child/run_plugins.js#L111

Reply:
But I should rename it in every file as well.

Reply:
done in 8909756479f6c57d4606e15c13b7023ea9ef8a16 & 5390b5d88461b2b948e0b100d4a911cc52bfd5f5